### DISCUSSION TITLE
Some URLs are not saved, no output in the console

### DISCUSSION THREAD

**Discussion by mariavd** [DominikPieper/obsidian-ReadItLater#146] [Q&A] [answered]
In some rare cases, the URL that I have copied doesn't get saved, and there is no error message. I have checked the Console and there is no sign of any messages. Do you have an idea what could be causing this? Here is an example:

https://emptyeasel.com/2017/01/23/50-ways-to-promote-your-art-and-engage-with-art-lovers/

**Comment by Nitero** [comment, marked as answer]
fix should be in https://github.com/DominikPieper/obsidian-ReadItLater/pull/163
